Description: Upstream changes introduced in version 3.2.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 packetfence (3.2.0-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
 .
 The person named in the Author field signed this changelog entry.
Author: Durand fabrice <fdurand@inverse.ca>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

+++ packetfence-3.2.0/addons/freeradius-integration/radiusd.conf.pf
@@ -2,23 +2,23 @@ prefix = /usr
 exec_prefix = /usr
 sysconfdir = /etc
 localstatedir = /var
-sbindir = /usr/sbin
-logdir = ${localstatedir}/log/radius
-raddbdir = ${sysconfdir}/raddb
+sbindir = ${exec_prefix}/sbin
+logdir = /var/log/freeradius
+raddbdir = /etc/freeradius
 radacctdir = ${logdir}/radacct
 
-name = radiusd
+name = freeradius
 
 confdir = ${raddbdir}
 run_dir = ${localstatedir}/run/${name}
 
 db_dir = ${raddbdir}
 
-libdir = /usr/lib64/freeradius
+libdir = /usr/lib/freeradius
 pidfile = ${run_dir}/${name}.pid
 
-user = radiusd
-group = radiusd
+user = freerad
+group = freerad
 
 max_request_time = 30
 cleanup_delay = 5
